mentii/mentii
# Html email

Sends an HTML email to users who register in the system.

**Significant Changes**
1. HTML email
2. Activation link is based on environment
3. Clicking on activation link brings them to a page on the Frontend that tells them if their account was successfully activated or not.

**How to Test**
1. Go to register page
2. Register for an account
3. Check email is HTML email
4. Check activation link reflects environment
5. Click on link and check that you see a happy face
6. Sign in with registered user
7. Copy past link into browser and remove part of the activation id
8. Check that you see a sad face
9. Throttle network speed and check that when activation link is clicked a loading screen is given.

**Notes**
:cactus:
